### ISSUE TITLE
Error: The project entry file could not be resolved.

### ISSUE THREAD

**Issue by slorber** [slorber/gatsby-plugin-react-native-web#21]
Now when usinjg the @expo/webpack-config, we get a warning

```
Error: The project entry file could not be resolved. Please either define it in the `package.json` (main), `app.json` (expo.entryPoint), create an `index.js`, or install the `expo` package.
```

A workaround is to just put an empty index.js at the root of your project.

This gatsby plugin (and thus the @expo/webpack-config) can be used with plain RNW. User might not always want to use Expo SDK web apis. 

I see 2 solutions:
- @expo/webpack-config should allow to bypass this warning (I guess it can make sense for NextJS integration too)
- This gatsby plugin could fallback to another webpack config if expo dependency is not found (seems not ideal, as expo config does some useful things we want to keep)

@EvanBacon any idea how this should be handled?

**Comment by felippepuhle** [comment]
it seems to be solved on `2.0.0-beta.0`

**Comment by ONeill45** [comment]
I encountered this error and was able to bypass using a blank index.js file in the project root. However I am now getting:

`Error: It appears like Gatsby is misconfigured. Gatsby related `graphql` calls are supposed to only be evaluated at compile time, and then compiled away. Unfortunately, something went wrong and the query was left in the compiled code.`

Has anyone seen this issue using this plugin?

**Comment by ONeill45** [comment]
Solved it using the solution offered by @slorber in #23

**Comment by slorber** [comment]
yeah sorry about that, I can't really make this automatic for now, need some modifications in expo webpack config

**Comment by slorber** [comment]
@felippepuhle 2.0 is more stable but has not the Expo webpack config so you'll have to hack yourself to make the project work if you are using external RNW compatible dependencies. Will try to make 3.0 more stable soon so that everything works fine out of the box.

**Comment by felippepuhle** [comment]
In fact I'm using `3.0.0-beta.0` and it's working with `babel-preset-expo`

**Comment by slorber** [comment]
Hi, just wanted to let you know that latest beta is likely more stable and you should not need to configure anything on your side.

**Comment by felippepuhle** [comment]
Thanks guys,  you're amazing! 😄

**Comment by EvanBacon** [comment]
I can work on a fix. What is the main entry point that Webpack should resolve to?
